Dockerfile

```diff
@@ -19,9 +19,9 @@ RUN if [ "$USE_CHINA_NPM_REGISTRY" = 1 ]; then \
 RUN npm i -g npm
 
 RUN if [ "$PUPPETEER_SKIP_CHROMIUM_DOWNLOAD" = 0 ]; then \
-  unset PUPPETEER_SKIP_CHROMIUM_DOWNLOAD && npm ci ;\
+  unset PUPPETEER_SKIP_CHROMIUM_DOWNLOAD && npm i ;\
   else \
-  export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true && npm ci ;\
+  export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true && npm i ;\
```

Comment:
Please pull the latest changes from master branch.
Building issue has been fixed in https://github.com/DIYgod/RSSHub/commit/b36b326742e18f1f1506c3a7c59c565dfb6f1b82